src/app/caesar-portal/caesar-login/caesar-login.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { NgModel } from '@angular/forms';
+import { CaesarLoginService } from './service/caesar-login.service';
+import { Router } from '@angular/router';
+
+@Component({
+    selector: 'caesar-login',
+    templateUrl: './caesar-login.component.html',
+    styleUrls: ['./caesar-login.component.scss']
+})
+export class CaesarLoginComponent implements OnInit {
+	data: any = {};
+
+    constructor (private router: Router,
+        private caesarLoginService: CaesarLoginService) { }
+
+    ngOnInit () {
+    }
+
+    keyPress (event: any): void {
+        let enterButton: number = 13,
+            escButton: number = 27;
+
+        if (event.keyCode === enterButton) {
```

Comment:
I did it at first but I realized that I need not only tracking pushing ‘enter’ but also ‘esc’.